app/styles/pages/events.scss

```diff
@@ -13,3 +13,7 @@
     width: 100%;
   }
 }
+
+.ui.column.grid>[class*="fifteen wide"].column, .ui.grid>.column.row>[class*="fifteen wide"].column, .ui.grid>.row>[class*="fifteen wide"].column, .ui.grid>[class*="fifteen wide"].column {
+  width: 100%!important;
+}
```

Comment:
No

Reply:
> No

So can you tell me what is the correct point to fixed this issue

Reply:
Why do you need media query?

Reply:
> So can you tell me what is the correct point to fixed this issue

Use only sematic UI while dealing with grids. No media queries.

Reply:
Sure

Reply:
Can you plz provide me the file name regarding this issue because i am new to  open source contribution can you sir help me?

Reply:
@iamareebjamal   
@maze-runnar 
can you plz review the  new changed

Reply:
@iamareebjamal  plz review the changed